src/mame/taito/taito_f3_v.cpp

```diff
@@ -384,10 +386,22 @@ TILE_GET_INFO_MEMBER(taito_f3_state::get_tile_info)
 	// This fixes (at least) the rain in round 6 of Arabian Magic.
 	const u8 extra_planes = ((tile >> (16 + 10)) & 3); // 0 = 4bpp, 1 = 5bpp, 2 = unused?, 3 = 6bpp
 
-	tileinfo.set(3,
-			tile & 0xffff,
-			(tile >> 16) & 0x1ff & (~extra_planes),
-			TILE_FLIPYX(tile >> 30));
+	// some games don't want this. maybe not bpp-based truncation?
+	// (landmakr win mes load to palette 0xBE, quizhuhu fade effects)
+	if (m_game == LANDMAKR || m_game == QUIZHUHU)
```

Comment:
Per-game hacks are unacceptable, this file already has too many of them. 
I'm suspecting the HW can really configure attribute bits to something that isn't color on-demand, and the Arabian Magic Rain "feature" above sounds like ... blinking!? (hard to say from the best video I've found: https://youtu.be/CCiLF5pKaS8?t=1329 )

Reply:
Forgot to check MT:
https://youtu.be/8ZXwVo2EisE?t=2241
https://mametesters.org/view.php?id=2733

Need to check bleeding edge up to stage 6 and side by side from this.

Reply:
i've looked at the rain, it really is one palette line off, but...
my guess is that palette lines should only be truncated to even lines, so 6bpp and 5bpp would have the same adjustment, the problem is that supposedly that original truncation fixed a bunch of other games that aren't noted in source that it means extensive testing: https://mametesters.org/view.php?id=1925 https://mametesters.org/view.php?id=1917 https://mametesters.org/view.php?id=900

really i wanted to address it in a follow-up because of the 'play an entire fighting game to check regression' condition w

Reply:
After some discussion from other devs I'm backing off my stance about my original comment here: I'd add these MT links to the original note about Arabian Magic rain, "at least" it's not a clever note imho and it's clearly `FIXME` league.